pulsar/internal/lookup_service.go

```diff
@@ -46,21 +46,25 @@ type LookupService interface {
 
 type lookupService struct {
 	rpcClient           RPCClient
-	serviceNameResolver ServiceNameResolver
+	serviceURL          *url.URL
```

Comment:
Why you change this? Look like we need the `ServiceNameResolver` to resolve the multiple service address.

Reply:
This is just to adjust the position when resolving the conflict, but not removed